.gitignore

```diff
@@ -39,3 +39,7 @@ yarn-error.log*
 # Miscelleanous
 
 .DS_Store
+
+# Other
+
+schema.prisma
```

Comment:
We're ignoring the `schema.prisma` file since we don't want to have to manage this file and continuously update it whenever the database changes, since it's not a critical workflow (ie: not used in our applications).